class NotFoundError extends AppError {
    constructor(resource = 'Resource') {
        super(`${resource} n√£o encontrado(a)`, 404);
    }
}

module.exports = NotFoundError;
